src/main/resources/digital/slovensko/autogram/server/server.yml

```diff
@@ -366,15 +366,27 @@ components:
           example: '<?xml version="1.0"?><xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"><xsl:template match = "/"><h1><xsl:value-of select="/Document/Title"/></h1></xsl:template></xsl:stylesheet>'
           description: "Optional XML transformation used to . Format is dictated by `payloadMimeType`."
 
+        previewWidth:
```

Comment:
Zacinaju sa nam tu miesat "parametre podpisu" a "pomocne parametre", nejako ma to nevyrusuje, ale toto API sa bude menit v buducnosti velmi tazko (spatna kompatiblita), pouvazujme silno, ze ci to nejako nechceme rozdelit.

Reply:
Hej, tiež sa mi to zdá rozumné rozdeliť. Napr už checkPDFA a táto prefWidth sú už dve veci. Popresúvam v API

Reply:
Pozor lebo uz aj extension to pdfa pouziva.

Reply:
@jsuchal rozmýšľal som nad tým ešte. V tom API by sme mohli oddeliť `signingParameters` alebo `signatureParameters` (všetky doteraz) a `otherParameters`. Tam by bola zatiaľ iba táto šírka alebo ešte aj to check PDFA. Tak či tak, pokazilo by to doterajšie API tým premenovaním.

Druhá možnosť je v existujúcich `paramters` pridať objekt `otherParamters` a do neho pridávať nové, ktoré sa nebudú priamo týkať podpisu. CheckPDFA by som nechal, kde sú, a teda nepokazilo by to už nič teraz.

Tretia možnosť je pridať `otherParamters` vedľa `parameters`.

Rozmýšľam, čo také ešte môžeme v budúcnosti posielať okrem iných rozmerov. Možno preferovaný podpisový driver, idk.

Potom hromadné podpisovanie by som asi spravil v osobitnom endpointe.

Reply:
hromadne podpisovanie bude urcite vlastny endpoint, tam musime kompletne pomenit logiku